e2e/playwright/info-drawer/src/tests/comments.spec.ts

```diff
@@ -0,0 +1,153 @@
+    await personalFiles.acaHeader.viewDetails.click();
+    await personalFiles.infoDrawer.commentsTab.click();
+    await expect(personalFiles.infoDrawer.commentInputField).toBeVisible();
+    expect(await personalFiles.infoDrawer.checkCommentsHeaderCount()).toEqual(0);
+    expect(await personalFiles.infoDrawer.addCommentButton).toBeDisabled();
+  });
+
+  test('[C299209] from Favorites - Add a comment on a folder', async ({ favoritePage }) => {
+    const folderFavId: string = (await nodesApi.createFolder(favFolderName)).entry.id;
+    await favoritesActions.addFavoritesByIds('folder', [folderFavId]);
+    await fileActionsApi.waitForNodes(favFolderName, { expect: 1 });
+    await favoritePage.navigate();
+    await expect(favoritePage.dataTable.getRowByName(favFolderName)).toBeVisible();
+    await favoritePage.dataTable.getRowByName(favFolderName).click();
+    await favoritePage.acaHeader.viewDetails.click();
+    await favoritePage.infoDrawer.commentsTab.click();
+    await expect(favoritePage.infoDrawer.commentInputField).toBeVisible();
+    await favoritePage.infoDrawer.addCommentToNode(commentText);
+    await expect(favoritePage.infoDrawer.addCommentButton).toBeDisabled();
+    expect(await favoritePage.infoDrawer.checkCommentsHeaderCount()).toEqual(1);
+    expect(await favoritePage.infoDrawer.verifyCommentsCountFromList(1));
+  });
+
+  test('[C299189] from Shared Files - Comments are displayed ordered by created date in descending order', async ({ sharedPage }) => {
+    const e2eCommentFirst = `e2e-comment-${Utils.random()}`;
+    const e2eCommentSecond = `e2e-comment-${Utils.random()}`;
+    const sharedFileId: string = (await nodesApi.createFile(sharedFileName)).entry.id;
+    await fileActionsApi.waitForNodes(sharedFileName, { expect: 1 });
+    await apiClientFactory.commentsApi.createComment(sharedFileId, { content: e2eCommentFirst });
+    await apiClientFactory.commentsApi.createComment(sharedFileId, { content: e2eCommentSecond });
+    await apiClientFactory.share.createSharedLink({ nodeId: sharedFileId });
+    await fileActionsApi.waitForNodes(sharedFileName, { expect: 1 });
+    await sharedPage.navigate();
+    await expect(sharedPage.dataTable.getRowByName(sharedFileName)).toBeVisible();
+    await sharedPage.dataTable.getRowByName(sharedFileName).click();
+    await sharedPage.acaHeader.viewDetails.click();
+    await sharedPage.infoDrawer.commentsTab.click();
+    await sharedPage.infoDrawer.waitForComments();
+    expect(await sharedPage.infoDrawer.checkCommentsHeaderCount()).toEqual(2);
+    expect(await sharedPage.infoDrawer.commentTextContent.nth(0).textContent()).toEqual(e2eCommentSecond);
+    expect(await sharedPage.infoDrawer.commentTextContent.nth(1).textContent()).toEqual(e2eCommentFirst);
+  });
+
+  test('[C299195] from Recent Files - Add a comment on a file', async ({ recentFilesPage }) => {
+    const recentFile = `e2e-recentfile-${Utils.random()}`;
```

Comment:
can you rename to `recentFile` to fix the linter warning

Reply:
What exactly should I rename to `recentFile`?

Reply:
Will be fixed in the next push